TransferMethod/Tests/SelectTransferMethodTypePresenterTests.swift

```diff
@@ -3,6 +3,7 @@ import Common
 #endif
 import Hippolyte
 import HyperwalletSDK
+import Insights
```

Comment:
Why do you need Insights?

Reply:
func trackError(pageName: String, pageGroup: String, errorInfo: ErrorInfo) {...

Reply:
func trackError(pageName: String, pageGroup: String, errorInfo: ErrorInfo) {... }
is from HyperwalletInsights that's in Common